cinema/views.py

```diff
@@ -37,12 +48,33 @@ class MovieViewSet(viewsets.ModelViewSet):
     def get_serializer_class(self):
         if self.action == "list":
             return MovieListSerializer
-
         if self.action == "retrieve":
             return MovieDetailSerializer
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        if self.action == ("list", "retrieve"):
```

Comment:
The condition `if self.action == ("list", "retrieve"):` is incorrect. The `self.action` attribute is a string, so it should be compared to a single string value, not a tuple. Consider using `if self.action in ["list", "retrieve"]:` to check if the action is either 'list' or 'retrieve'.